tasks/collect-collectors-data/README.md

```diff
@@ -0,0 +1,19 @@
+# collect-collectors-data
+
+Tekton task to collect the information related to collectors.
+
+The purpose of this task is to collect all the data and supply it to other tasks in the pipeline by creating
```

Comment:
I read this and thought okay he is going to fetch the collectors field from each CR. That is not what is happening, so I don't think this description is accurate. The task as written fetches all CRs that relate to collectors. The "data" string doesn't even exist in the task yaml, and it has special meaning for us so I don't think it should be talked about here if it isn't touched. In fact, I don't even like the word data in the task name given this. `collect-collectors-resources` or something

Reply:
Renamed to collect-collectors-resources.